### ISSUE TITLE
Can't check in files with no changes

### ISSUE THREAD

**Issue by Ameliarate16** [ProjectBorealis/UE4GitPlugin#148]
My team has been using this plugin for a project but we've been having an issue where some files are checked out but the check in button is greyed out and there's no saved changes to submit. We tried manually unlocking the files in git bash but it said "git status failed: exited with code 128" which is the generic 'raised an error' code which was no help.

**Comment by nshkurkin** [comment]
Did you ever find a resolution or workaround to this? Just integrated into our fresh UE5 project and seeing something similar where if you cancel out of a Check In operation on an unchanged file it looks as if you no longer have the file locked and can't Revert/CheckIn anymore. Going to the shell and using `git lfs unlock <path>` worked for us though.

**Comment by Ameliarate16** [comment]
Nope. We ended up just removing the plugin and unlocking the files from command line.

**Comment by mastercoms** [comment]
I've had issues reproducing this, unfortunately
